### PR TITLE
Dev: Extra tags in media query for better styling on mobile

### PR DESCRIPTION
**Pull request by tammoterhark**
@shnoulle please reviw the latest commit. The others: I'm sorry; they do something and after that revert it. I am afraid to loose work if I delete these...

### CODE REVIEW

File: templates/default/css/template.css [hunk 1]
```diff
@@ -622,3 +624,47 @@ div.yes-no .btn-group label {
...
+    border:none;
+  }
+  
+  .ls-answers tr:last-child{
+    margin-bottom:0;
+  }
+  
+  .ls-answers > tbody > tr > td.radio-item ,
+  .ls-answers > tbody > tr > td.checkbox-item{
+    min-height:44px; /*minimal height for touching with finger */
```
Comment (Shnoulle) [inline templates/default/css/template.css]: In this case : max-device-width: and not width for media

File: templates/default/css/template.css [hunk 2]
```diff
@@ -622,3 +624,47 @@ div.yes-no .btn-group label {
...
 }
+
+@media only screen and (max-width: 760px){
+  h1, .h3{
+    font-size:18px;
+  }
+  
+  .h4{
+    font-size:16px;
+  }
```
Comment (Shnoulle) [inline templates/default/css/template.css]: Unsure for this part :/

File: templates/default/css/template.css [hunk 3]
```diff
@@ -622,3 +624,47 @@ div.yes-no .btn-group label {
...
+  
+  .ls-answers tr:last-child{
+    margin-bottom:0;
+  }
+  
+  .ls-answers > tbody > tr > td.radio-item ,
+  .ls-answers > tbody > tr > td.checkbox-item{
+    min-height:44px; /*minimal height for touching with finger */
+    padding-top:10px;
+    padding-left:28px;
```
Comment (Shnoulle) [inline templates/default/css/template.css]: OK for padding top , but padding-left must be same than li in big screen, no ?

File: templates/default/css/template.css (hunk 1 above)
Comment (tammoterhark) [inline templates/default/css/template.css]: Sorry, do not understand your comment.

File: templates/default/css/template.css [hunk 4]
```diff
@@ -89,12 +96,18 @@ padding-top:1em;padding-bottom:1em;margin-bottom:2em;
 .answer-container {
   border-width: 0px 1px 0px 1px;
   border-style: solid;
-  padding-top: 3em;
-  padding-bottom:1.5em;
+  //padding-top: 3em;
+  //padding-bottom:1.5em;
```
Comment (Shnoulle) [inline templates/default/css/template.css]: Really NEED a margin here !

File: templates/default/css/template.css (hunk 2 above)
Comment (Shnoulle) [inline templates/default/css/template.css]: I don't like it ..... to little.

File: templates/default/css/template.css (hunk 1 above)
Comment (Shnoulle) [inline templates/default/css/template.css]: width : big screen with little windows max-device-width : phone or really little screen

File: templates/default/css/template.css (hunk 4 above)
Comment (Shnoulle) [inline templates/default/css/template.css]: https://github.com/LimeSurvey/LimeSurvey/commit/040db0d38c48e2403f87adb0f1a16825995ea74f

File: templates/default/css/template.css (hunk 3 above)
Comment (tammoterhark) [inline templates/default/css/template.css]: You are right, should be the same in big screen. Shal I change that? Please accept the current changes and I will do it again. (I do not know how do retract commits...)

File: templates/default/css/template.css (hunk 4 above)
Comment (tammoterhark) [inline templates/default/css/template.css]: Please look, I added it in another class.

Comment (Shnoulle) [inline templates/default/css/template.css]: before my commit : no space before/after the ls-answers (already merged)

File: templates/default/css/template.css (hunk 1 above)
Comment (tammoterhark) [inline templates/default/css/template.css]: Do you mean that I should change the mediaquery?

File: templates/default/css/template.css (hunk 2 above)
Comment (tammoterhark) [inline templates/default/css/template.css]: Will you do that or shall I do it?

Comment (Shnoulle) [inline templates/default/css/template.css]: Like you want :) i think i try to fix the bug for array by dual scale (https://bugs.limesurvey.org/view.php?id=11863) before

File: templates/default/css/template.css (hunk 4 above)
Comment (tammoterhark) [inline templates/default/css/template.css]: You put in margin, while it should have been padding. Look at box around answers..

Comment (Shnoulle) [inline templates/default/css/template.css]: Or, margin on ls-answers ? But littlest than before.

**Pull request merged**